diff --git a/build/src/start/platforms/android/adb.js b/build/src/start/platforms/android/adb.js
index 4672d4b43ebafb091dfaa60e7bdd296a8e2db693..faf92f014c8c236ef07e3229ad18f5e75280a3a0 100644
--- a/build/src/start/platforms/android/adb.js
+++ b/build/src/start/platforms/android/adb.js
@@ -210,9 +210,9 @@ async function installAsync(device, { filePath }) {
 }
 function adbArgs(pid, ...options) {
     const args = [];
-    if (pid) {
-        args.push('-s', pid);
-    }
+    // if (pid) {
+    //     args.push('-s', pid);
+    // }
     return args.concat(options);
 }
 function adbShellArgs(pid, ...command) {
